kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -48,6 +48,7 @@ class Foundry:
     _root: Path
     _toml: dict[str, Any]
     _bug_report: BugReport | None
+    _requires: Iterable[str] | None
```

Comment:
Why do we have this and the property `requires`? Can we just have the field named `requires` then directly?

Reply:
There is just a field now.